packages/dashboard/src/components/iot-dashboard/iot-dashboard-internal.tsx

```diff
@@ -419,6 +456,19 @@ export class IotDashboardInternal {
             }}
           ></div>
         )}
+
+        <iot-dashboard-context-menu
+          messageOverrides={this.messageOverrides}
+          actions={{
+            onCopy: this.onCopy.bind(this),
+            onPaste: this.onPaste.bind(this),
```

Comment:
skip the binding and just define the methods as arrow functions so `this` is always lexical